trunk/NDHMS/template/NoahMP/namelist.hrldas

```diff
@@ -1,72 +1,73 @@
- TEMP_TIME_SCHEME_OPTION           = 3
- GLACIER_OPTION                    = 2
- SURFACE_RESISTANCE_OPTION         = 4
+! Physics options (see the documentation for details)
+DYNAMIC_VEG_OPTION                = 4
+CANOPY_STOMATAL_RESISTANCE_OPTION = 1
+BTR_OPTION                        = 1
+RUNOFF_OPTION                     = 3
+SURFACE_DRAG_OPTION               = 1
+FROZEN_SOIL_OPTION                = 1
+SUPERCOOLED_WATER_OPTION          = 1
+RADIATIVE_TRANSFER_OPTION         = 3
+SNOW_ALBEDO_OPTION                = 2
+PCP_PARTITION_OPTION              = 1
+TBOT_OPTION                       = 2
+TEMP_TIME_SCHEME_OPTION           = 3
+GLACIER_OPTION                    = 2
+SURFACE_RESISTANCE_OPTION         = 4
 
- ! Timesteps in units of seconds
- FORCING_TIMESTEP = 3600 
- NOAH_TIMESTEP    = 3600 
- OUTPUT_TIMESTEP  = 3600 
+! Timesteps in units of seconds
+FORCING_TIMESTEP = 3600
+NOAH_TIMESTEP    = 3600
+OUTPUT_TIMESTEP  = 3600
 
- ! Land surface model restart file write frequency
- RESTART_FREQUENCY_HOURS = 2
+! Land surface model restart file write frequency
+RESTART_FREQUENCY_HOURS = 2
 
- ! Split output after split_output_count output times.
- SPLIT_OUTPUT_COUNT = 1
+! Split output after split_output_count output times.
+SPLIT_OUTPUT_COUNT = 1
 
- ! Soil layer specification
- NSOIL=4
- soil_thick_input(1) = 0.10
- soil_thick_input(2) = 0.30
- soil_thick_input(3) = 0.60
- soil_thick_input(4) = 1.00
+! Soil layer specification
+NSOIL=4
+soil_thick_input(1) = 0.10
+soil_thick_input(2) = 0.30
+soil_thick_input(3) = 0.60
+soil_thick_input(4) = 1.00
 
- ! Forcing data measurement height for winds, temp, humidity
- ZLVL = 10.0
+! Forcing data measurement height for winds, temp, humidity
+ZLVL = 10.0
+
+! Restart file format options
+rst_bi_in = 0      !0: use netcdf input restart file
+                   !1: use parallel io for reading multiple restart files (1 per core)
+rst_bi_out = 0     !0: use netcdf output restart file
+                   !1: use parallel io for outputting multiple restart files (1 per core)
 
- ! Restart file format options
- rst_bi_in = 0      !0: use netcdf input restart file
-                    !1: use parallel io for reading multiple restart files (1 per core)
- rst_bi_out = 0     !0: use netcdf output restart file
-                    !1: use parallel io for outputting multiple restart files (1 per core)
-/
 
 &WRF_HYDRO_OFFLINE
 
- ! Specification of forcing data:  1=HRLDAS-hr format, 2=HRLDAS-min format, 3=WRF, 
- !                                 4=Idealized, 5=Ideal w/ spec. precip, 
- !                                 6=HRLDAS-hr format w/ spec. precip,
- !                                 7=WRF w/ spec. precip 
- FORC_TYP = 1
+! Specification of forcing data:  1=HRLDAS-hr format, 2=HRLDAS-min format, 3=WRF,
+!    4=Idealized, 5=Idealized w/ Spec. Precip.,
+!    6=HRLDAS-hourly fomat w/ Spec. Precip., 7=WRF w/ Spec. Precip.,
+!    9=Channel-only forcing, see hydro.namelist output_channelBucket_influxes
```

Comment:
Do we have an option 8?

Also, do we need disclaimers for option 9 and 10?  Maybe they're in the hydro.namelist?

Reply:
disclaimers? you mean "see hydro.namelist... "

Reply:
I'm good w/ it if you are.